data/uhc_pack/functions/running/diamond_chestplate/cleanup.mcfunction

```diff
@@ -0,0 +1,4 @@
+clear @s minecraft:diamond_chestplate
```

Comment:
Technically, there is a race condition between when you check for the diamond chestplate and you give the diamonds. This means that if you manage to pickup and immediately throw the diamond chest plate the next tick on the floor, you can duplicate the diamonds. Although this is extremely unlikely, I would still like to see this fixed. You can achieve this by changing this to 

```
execute store success score @s uhcDiaChest run clear @s minecraft:diamond_chestplate
execute if score @s uhcDiaChest matches 1 run give @s minecraft:diamond 8
```